clusterloader2/testing/density/config.yaml

```diff
@@ -137,6 +137,13 @@ steps:
     Method: SchedulingThroughput
     Params:
       action: gather
+      {{if ge .Nodes 500}}
+      # Setting to 90 should allow us to catch regressions like
+      # https://github.com/kubernetes/kubernetes/pull/85030 while not making the tests flaky.
+      # We do that only for 500+ node cluster as in smaller clusters the scheduler QPS may be
+      # less than 100 and hence the 100 throughput won't be achievable.
+      threshold: 90
+      {{end}}
```

Comment:
I'm not sure I like it - this is super hacky...
And we actually bump it for 100-node clusters too on GCE.

Reply:
Why it's hacky? IMHO, it's reasonable to assume that certain throughput will be achievable only in large enough clusters
I agree that we won't be alerted on regressions in  100 node gce clusters, but this is still way better than what we currently have  = no alerts at all.

Reply:
The alternative is to gate this by provider which imho is more hacky. I don't see anything hacky in saying that we enforce scheduler throughput in clusters having at least X nodes. If you don't like  X=500 then I'm open for discussion :)

Reply:
I don't like baking assumptions about providers (it is kind-of provider-specific logic) into our generic tests.
Note that, we actually set qps=100 in e2e-common preset:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml#L87

Rather than that, I would prefer making a threshold here some kind of parameter, and baking explicit thesholds into test definitions (overrides) used explicitly in tests we want to test against it.

Reply:
I like the new approach - thanks.